articles/active-directory/develop/scenario-protected-web-api-app-configuration.md

```diff
@@ -240,4 +240,4 @@ You can also validate incoming access tokens in Azure Functions. You can find ex
 ## Next steps
 
 Move on to the next article in this scenario,
-[Verify scopes and app roles in your code](scenario-protected-web-api-verification-scope-app-roles.md).
\ No newline at end of file
+[Verify scopes and app roles in your code](scenario-protected-web-api-verification-scope-app-roles.md).
```

Comment:
what is this change?

Reply:
I don't know what that change is... It somehow go included and I can't see the difference. Maybe something encoding related?